### ISSUE TITLE
Android - HLS video - Once reproduced, video freeze.

### ISSUE THREAD

**Issue by eleva** [nchutchind/cordova-plugin-streaming-media#77]
Latest version of this plugin.
Cordova Android 5.1.1.
Samsung S6 Edge Plus, Samsung S4, Samsung S3, Motorola Moto X.

It seems that on Android (I tested it on multiple device), once the video is fully played one time, it completely freezes. It could not be played again (neither clicking on play button or moving the video time slider) and video time is frozen at 0.00.

What is expected is that we could play agin the video, video time slider is still working and video time goes to the actual length of the video (or the actual time of the video playing). 

Here is the code affected. 
Simply tring to reproduce a HLS video, serving it from amazon s3.

`document.addEventListener("deviceready", function() {
  var options = {
    successCallback: function() {
      console.log("Video was closed without error.");
    }, 
    errorCallback: function(errMsg) {
      console.log("Error! " + errMsg);
    },
    orientation: 'portrait'
  };
  window.plugins.streamingMedia.playVideo('https:/example/example/example.m3u8', options);
}, false);`

**Comment by xiejiabao** [comment]
@eleva Hi, did you find any solution or alternative?

**Comment by xjeeh** [comment]
In my case the HLS videos are crashing the player in iOS after system update (ios103). Normal videos are running okay. Is anyone facing this problem as well?

**Comment by satishSKY** [comment]
Yes, i am also facing same issue, player can not work with hls video url.
any one can solve this ?

**Comment by fdambrosio** [comment]
How can I play HLS streaming on Android/iOS using Cordova/Ionic platform?

**Comment by nystsam** [comment]
I'm facing this issue too, is there any solution?

**Comment by fdambrosio** [comment]
@nystsam  I'm using cordova-plugin-media, it's working with HLS

**Comment by nystsam** [comment]
Yeah, but that plugin is for audio recording, not videos

**Comment by jayarjo** [comment]
So playing HLS on Android/iOS using Cordova/Ionic not possible?

**Comment by fdambrosio** [comment]
@jayarjo yes it's possible on Ionic, for hls video playing I'm using videogular

**Comment by shamilovtim** [comment]
There are other plugins (as well as native browser scripts like Videogular) which support HL. It's just not a feature this current plugin has

**Comment by jayarjo** [comment]
So the point is to use any lib with HLS support and that will do?
